reportportal/values.yaml

```diff
@@ -103,6 +106,7 @@ serviceui:
   serviceAccountName: ""
   service:
     portName: ""
+  
```

Comment:
nitpitck: Can probably not add this extra blank line, rest of it looks decent to eyeballs.

Reply:
Removed the extra blank line.